arraykit.c

```diff
@@ -351,6 +371,7 @@ static PyMethodDef ArrayGO_methods[] = {
     {"append", (PyCFunction)ArrayGO_append, METH_O, NULL},
     {"copy", (PyCFunction)ArrayGO_copy, METH_NOARGS, ArrayGO_copy_doc},
     {"extend", (PyCFunction)ArrayGO_extend, METH_O, NULL},
+    {"__getnewargs__", (PyCFunction)ArrayGO_getnewargs, METH_NOARGS, NULL},
```

Comment:
Also, can you add a stub for this method in `arraykit.pyi`?

Reply:
Thanks for the reminder!